macrosynergy/visuals/README.md

```diff
@@ -0,0 +1,186 @@
+# macrosynergy.visuals
```

Comment:
Secondary README - expand on the root/README.md as well with description of the plotting functionality and how to call it.